docs/blog/2020-02-18-gazette-2019-recap/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Gatsby Gazette: The 2019 Recap"
+date: 2020-02-18
+author: Sidhartha Chatterjee
+excerpt: "The last year has been an exciting time for Gatsby. Let's recap over everything we did in 2019."
+tags:
+  - gazette
+  - community
+---
+
+Welcome to a _special_ edition of the Gatsby Gazette. This one will recap everything we did in 2019. Well, not _everything_. But everything major enough that we can fit into this post.
+
+For our more frequent readers who might already know much of this, we'll follow up with a walkthrough of our plans for 2020.
+
+Before we dive in, thank you, dear reader. If you've been contributing code to Gatsby, helping improve our documentation, reporting bugs and issues or even simply using it, you've enabled so much of the progress we've made and will continue to make this year. Our superpower is _you_, the Gatsby community.
+
+## The 2019 Recap
+
+Let's get into it then.
+
+1348 folks started contributing to Gatsby in 2019.
+
+Together, we made 5657 commits to Gatsby in 2019.
+
+That's a 29% increase from 2018 (4364) and a 153% increase from 2017 (2236).
+
+We released Gatsby 467 times in 2019. 😱 (Run `npm view gatsby time` to display the full list)
+
+Out of these, 18 were minor releases.
+
+### 2.1.0: useStaticQuery hook
+
+(2019-02-13)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.120...gatsby@2.1.0)
+
+The `useStaticQuery` hook was the first minor release in 2019. The author of this post built this just in time for the first stable release of React Hooks.
+
+### 2.2.0: Schema Customisation
+
+(2019-03-19)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.1.39...gatsby@2.2.0)
+
+This was a huge milestone in the development of Gatsby's GraphQL data layer. Built by Mikhail and Stefan, this enabled users to declare their own custom GraphQL schema. Before this, you site's schema was exclusively inferred by Gatsby leading to a dependency on data and making it brittle.
```

Comment:
Not commenting every occurrence so valid for whole document:

Should we link the names to GitHub/Twitter profiles? As cool as e.g. Mikhail and Stefan are, they are not _that_ famous yet.

Reply:
Haha, yup. Need to link folks to Twitter and feature titles to those diff urls